source/SelectStationMenuDelegate.mc

```diff
@@ -3,15 +3,23 @@ import Toybox.Application.Storage;
 
 using Toybox.WatchUi;
 
+// TODO: can we possibly just roll this into the LoadMoreMenuDelegate?
```

Comment:
It would be possible to simply roll this into the LoadMoreMenuDelegate; however, that may add even more complexity to that class, and also complicate sitauations where we only need this simple menu delegate.